updater/spec/dependabot/api_client_spec.rb

```diff
@@ -101,6 +101,34 @@
             end)
       end
     end
+
+    context "with a removed dependency" do
```

Comment:
What happens if somehow we pass in `removed: false`?

Reply:
It defaults to false so should be the same: https://github.com/dependabot/dependabot-core/blob/main/common/lib/dependabot/dependency.rb#L42-L44